src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,75 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../contexts/TodosContext';
+import { haveAnyCompletedTodo } from '../../services/todo';
+import { Status } from '../../types/Status';
+
+export const TodoFooter: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    filterStatus,
+    setFilterStatus,
+  } = useContext(TodosContext);
+
+  const activeTodos = todos.filter(todo => !todo.completed);
+
+  const handleClearingCompleted = () => {
+    setTodos(activeTodos);
+  };
+
+  return (
+    <footer className="footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${activeTodos.length} items left`}
+      </span>
+
+      <ul className="filters">
+        <li>
+          <a
+            href="#/"
+            className={filterStatus === Status.All
+              ? 'selected'
+              : ''}
+            onClick={() => setFilterStatus(Status.All)}
+          >
+            All
+          </a>
+        </li>
+
+        <li>
+          <a
+            href="#/active"
+            className={filterStatus === Status.Active
+              ? 'selected'
+              : ''}
+            onClick={() => setFilterStatus(Status.Active)}
+          >
+            Active
+          </a>
+        </li>
+
+        <li>
+          <a
+            href="#/completed"
+            className={filterStatus === Status.Completed
```

Comment:
It is advisable to prevent the default event in the onClick handler to avoid reloading the page.